### DISCUSSION TITLE
What data was this model trained with? Specifically the NER?

### DISCUSSION THREAD

**Discussion by owquresh** [microsoft/presidio#1180] [Design records]
Want to see what dataset the model was trained on, mainly the named entity recognition that's going on, is there a repo for that?

**Comment by omri374** [comment]
Presidio uses open-source models. The default one used is [`en_core_web_lg`](https://spacy.io/models/en#en_core_web_lg) which was trained on the [OntoNotes 5 corpus](https://catalog.ldc.upenn.edu/LDC2013T19). Most common datasets are OntoNotes 5, CONLL2003, I2B2 2006, I2B2 2014, and a combination of those with synthetic data.
